src/fides/api/app_setup.py

```diff
@@ -67,12 +66,12 @@
 DEFAULT_PRIVACY_NOTICES_PATH = join(
     dirname(__file__),
     "../data/privacy_notices",
-    "privacy_notice_templates.yml",
+    "privacy_notice_templates2.yml",
 )
 PRIVACY_EXPERIENCE_CONFIGS_PATH = join(
     dirname(__file__),
     "../data/privacy_notices",
-    "privacy_experience_config_defaults.yml",
+    "privacy_experience_config_defaults2.yml",
```

Comment:
This may or may not be straightforward as to what this looks like.  For the "generic" notices I ended up having to create a new out of the box Experience Config to preserve the same options that existed when regions + displayed_in* were stored on the notice.

Reply:
Working on a draft of this now want to get ahead of it